packages/vue-query/tsconfig.json

```diff
@@ -4,10 +4,9 @@
     "composite": true,
     "rootDir": "./src",
     "outDir": "./build/lib",
-    "tsBuildInfoFile": "./build/.tsbuildinfo"
+    "tsBuildInfoFile": "./build/.tsbuildinfo",
+    "module": "esnext"
```

Comment:
doesn't this change the output of the type definitions for the `vue` package?

Reply:
@ZackDeRose Could you please explain why this change is necessary?

Reply:
Hey folks - yes I needed top-level await for those mock files. The equivalent of jest's synchronous `requireActual` is vitest's async `importActual`. I'll revert this and see if I can fix those mocks some other way.

Reply:
Looks like the error I was getting is only in the ts language server, so this is safe to revert. See comment in the mock file!